back-end/controllers/events.js

```
@@ -0,0 +1,30 @@
+const {request} = require('./axios');
+
+const acceptPending = (req, res) => {
+    request()
+    .post('/events.json')
+    .then((data) => {
+      res.status(200).json({'message': "event accepted", "event": data});
+    })
+    .catch((e) => {
+      console.error(e);
```

this catch should return some local mock data, because mockaroo only allows up to 200 requests, so if it fails, it means we cannot fetch from it anymore.